packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -196,324 +134,76 @@ class FlutterDriver {
-  /// useful if your application spawns multiple isolates that you
-  /// would like to instrument.
-  final VMServiceClient serviceClient;
-
-  /// JSON-RPC client useful for sending raw JSON requests.
-  rpc.Peer _peer;
-
-  String _dartVmReconnectUrl;
-
-  Future<void> _restorePeerConnectionIfNeeded() async {
-    if (!_peer.isClosed || _dartVmReconnectUrl == null) {
-      return;
-    }
-    _log('Peer connection is closed! Trying to restore the connection...');
-    final String webSocketUrl = _getWebSocketUrl(_dartVmReconnectUrl);
-    final WebSocket ws = await WebSocket.connect(webSocketUrl);
-    ws.done.whenComplete(() => _checkCloseCode(ws));
-    _peer = rpc.Peer(
-        IOWebSocketChannel(ws).cast(),
-        onUnhandledError: _unhandledJsonRpcError,
-    )..listen();
-  }
-
-  /// The main isolate hosting the Flutter application.
-  ///
-  /// If you used the [registerExtension] API to instrument your application,
-  /// you can use this [VMIsolate] to call these extension methods via
-  /// [invokeExtension].
-  final VMIsolate appIsolate;
-
-  /// Whether to print communication between host and app to `stdout`.
-  final bool _printCommunication;
-
-  /// Whether to log communication between host and app to `flutter_driver_commands.log`.
-  final bool _logCommunicationToFile;
-
-  Future<Map<String, dynamic>> _sendCommand(Command command) async {
-    Map<String, dynamic> response;
-    try {
-      final Map<String, String> serialized = command.serialize();
-      _logCommunication('>>> $serialized');
-      final Future<Map<String, dynamic>> future = appIsolate.invokeExtension(
-        _flutterExtensionMethodName,
-        serialized,
-      ).then<Map<String, dynamic>>((Object value) => value as Map<String, dynamic>);
-      response = await _warnIfSlow<Map<String, dynamic>>(
-        future: future,
-        timeout: command.timeout ?? kUnusuallyLongTimeout,
-        message: '${command.kind} message is taking a long time to complete...',
-      );
-      _logCommunication('<<< $response');
-    } catch (error, stackTrace) {
-      throw DriverError(
-        'Failed to fulfill ${command.runtimeType} due to remote error',
-        error,
-        stackTrace,
-      );
-    }
-    if (response['isError'] as bool)
-      throw DriverError('Error in Flutter application: ${response['response']}');
-    return response['response'] as Map<String, dynamic>;
   }
 
-  void _logCommunication(String message) {
-    if (_printCommunication)
-      _log(message);
-    if (_logCommunicationToFile) {
-      final f.File file = fs.file(p.join(testOutputsDirectory, 'flutter_driver_commands_$_driverId.log'));
-      file.createSync(recursive: true); // no-op if file exists
-      file.writeAsStringSync('${DateTime.now()} $message\n', mode: f.FileMode.append, flush: true);
-    }
-  }
+  /// Sends [command] to the Flutter Driver extensions.
+  /// This must be implemented by subclass. Check vmservice_driver and
```

Comment:
Use the 'see also' style from elsewhere in the tool: https://github.com/flutter/flutter/blob/449f4a6673f6d89609b078eb2b595dee62fd1c79/packages/flutter/lib/src/cupertino/icons.dart#L24